subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -129,6 +129,13 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
 
         then:
         scanRunner()
+            .expectLegacyDeprecationWarningIf(versionNumber < FIRST_VERSION_CALLING_BUILD_PATH,
+                "Usage of this Gradle Enterprise plugin version has been deprecated. " +
```

Comment:
Is it possible to print the version of the currently applied Gradle Enterprise plugin version?

Reply:
I added the version info to the deprecation warning. I think it makes sense